### PR TITLE
wqt create complete

### PR DESCRIPTION
**Pull request by dhillondeep**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,133 @@
+.idea/
```
Comment (Mogball) [inline .gitignore:1]: wth is this massive gitignore

**Review by Mogball — COMMENTED** [review]
Here's a fun suggestions, how about we build one massive toolchain that works for Cosa, Qt, general C++, and other C++ stuff

Or, we build a base toolchain with common functions for all of these, and then maintain specialized ones for each of the above

**Comment by dhillondeep** [comment]
Lool that will be huge but I think they should be separate because otherwise they will do too many things but actually if I really think about it, it is a good idea. Another side project :)

File: .gitignore (hunk 1 above)
Comment (dhillondeep) [reply .gitignore:1]: Lmao wanna be safe boy

**Review by Mogball — APPROVED** [review]
well I have no vested interest in this so I'll say lgtm because code seems similar to wcosa xd

**Pull request merged**